pyfakefs/fake_pathlib.py

```diff
@@ -521,18 +521,18 @@ def __new__(cls, *args, **kwargs):
                 else FakePathlibModule.PosixPath
             )
         if sys.version_info < (3, 12):
-            return cls._from_parts(args)
+            return cls._from_parts(args)  # pytype: disable=attribute-error
```

Comment:
`pathlib` has changed changed the design several times, so there is quite some code here related only to some Python versions